views/institutional/components/hero/index.tsx

```diff
@@ -9,7 +9,50 @@ const Hero: FC = () => (
     backgroundPosition="bottom center"
     backgroundSize="cover"
   >
-    <Box mt="2xl">Hero</Box>
+    <Box
+      height="100vh"
+      width="100vw"
+      overflow="hidden"
+      bg="url(/img/hero-illustration.svg)"
```

Comment:
try convert to `webp`

Reply:
Yes, I try to use webp but I don't like the image quality.
 SVG has a better quality than webp.